db-async-common/src/main/java/com/github/jasync/sql/db/pool/ObjectAgedOutException.kt

```diff
@@ -0,0 +1,4 @@
+package com.github.jasync.sql.db.pool
+
+class ObjectAgedOutException(obj: PooledObject, age: Long, maxTtl: Long) :
```

Comment:
This exception is not necessary in my opinion

Reply:
It exists only so I can hook into the code that calls validate, since they call dispose on exception.

Reply:
see the comment above